SConstruct

```diff
@@ -1145,6 +1145,20 @@ def CheckModuleLibs(config,try_libs,source_file,name,prepend=True):
     return result
 
 
+def GetPythonVersion(config):
+    # Get the version:
+    source_file5 = "import sys; print('%d.%d'%(sys.version_info[:2]))"
+    result, py_version = TryScript(config,source_file5,python)
+    # If that didn't work, try to get it from the file or directory names, since it is usually
+    # there:
+    if not result:
+        py_version = ''
+        for v in ['2.7', '2,6', '3.4', # supported versions first
+                  '2.5', '2,4', '3,5', '3.3', '3.2', '3.1', 3.0]:
```

Comment:
Yes, about `'3.0'`.  And these are just for determining what the version is, not a promise to support it, so I tried to include anything that would be remotely plausible that someone might have for their python distro.  If it doesn't find any of these, then py_version will just be `''`.